trackintel/io/file.py

```diff
@@ -337,10 +326,6 @@ def read_trips_csv(*args, columns=None, tz=None, index_col=None, geom_col=None,
     """
     Read trips from csv file.
 
```

Comment:
Yes to be consistent with the others this is a good idea.
I dropped the extracts proper datetimes description as pandas read_csv does that on its own.

Reply:
Ok but if pandas does it on its own, then do we still need the lines below: 

```python
trips["started_at"] = pd.to_datetime(trips["started_at"])
trips["finished_at"] = pd.to_datetime(trips["finished_at"])
```

So as long as its in the code I would suggest one of these three options: 
- leave it in the docstring
- delete it from the code
- none of the above but open an issue that this can be deleted later

Reply:
The code example is from line 385 in the new version of  `trackintel/io/file.py` in the `read_trips_csv` function

Reply:
Hm, yes I see your point.
But I think that the docstring `extracts proper datetime` is a bit ambigous anyway.
Also as the function returns a `Trips` class in this case, it must have "finished_at" and "started_at" datetime columns anyway.

Reply:
So I did the standard tactics, I waited long enough that I don't care anymore. I added the extra stuff in the documentation. :D